eplus.py

```diff
@@ -79,6 +178,11 @@ def open(self):
 
         self.writer.start()
         self.worker.start()
+        self.session_updater.start()
```

Comment:
The overridden `open` method can also be simplified now, since we no longer need a custom worker and since the ringbuffer and writer/worker init methods have been moved into `__init__()` in the latest streamlink:
https://github.com/streamlink/streamlink/blob/b277114d53ad07abe079283df738d6d6ae2746f8/src/streamlink/stream/segmented.py#L209

So we should now just need something like:

```py
def open(self):
    super().open()
    self.session_updater.start()
```

Reply:
Got it. However, as I mentioned earlier, we still need a custom worker, so I think it's better to put this change into a future PR regarding simplification and replacement of deprecated parts. Your opinion?

Reply:
I'm fine with handling it in a follow up PR